spec/system/task_spec.rb

```diff
@@ -100,13 +98,12 @@
   describe 'Task削除' do
     context '正常系' do
       # FIXME: テストが失敗するので修正してください
-      xit 'Taskが削除されること' do
-        project = FactoryBot.create(:project)
+      fit 'Taskが削除されること' do
```

Comment:
fitとしたままだとこのテストしか実行されなくなるのでご注意ください。
```suggestion
      it 'Taskが削除されること' do
```

Reply:
5f10a69